ts/src/binance.ts

```diff
@@ -556,7 +556,6 @@ export default class binance extends Exchange {
                         'ticker/price': { 'cost': 1, 'noSymbol': 2 },
                         'ticker/bookTicker': { 'cost': 1, 'noSymbol': 2 },
                         'openInterest': 1,
-                        'pmExchangeInfo': 1,
```

Comment:
@sc0Vu where did you see that this endpoint was deprecated? can't find it

Reply:
@carlosmiei 
fapi: https://binance-docs.github.io/apidocs/futures/en/#change-log
dapi: https://binance-docs.github.io/apidocs/delivery/en/#change-log

Although the dapi one still work, I think they'll removed it.

Reply:
@sc0Vu got it, yeah I think we can remove it